### DISCUSSION TITLE
Keyboard / login form size editable

### DISCUSSION THREAD

**Discussion by robbudge** [frangoteam/FUXA#1117] [Ideas]
is it possible to add the form size to be adjustable from within the editor ?
especially for login ?
we typically use at least a 1920 x 1080 screen and the 250px form used for login results in a very small target area for the operators fat fingers.

I typically edit login.component.html and rebuild
<div style="width: 250px;position: relative;padding-bottom: 40px"> and make the width 500px 
Or find it in the client main.js and modify.

It would really good if this could be set from the editor depending the screen size and resolution.
I suspect it would be relatively simple to implement but I only know enough to be dangerous.
I did see another post with some else commenting on the size of the virtual keyboard.

**Comment by unocelli** [comment]
Hi, you can use the full screen option
![image](https://github.com/frangoteam/FUXA/assets/48684550/b5a054e2-e0ab-4a6d-afa3-00c32ef0bfc6)
